### DISCUSSION TITLE
[Idea] Agent 智能助理的演进要素

### DISCUSSION THREAD

**Discussion by arvinxx** [lobehub/lobehub#552] [Ideas]
除了通过工具体系来丰富助理的能力，从长期来看 Agent 的构建应该包括以下要素：
- [x] 思考(大模型)
- [x] 多模态交互能力(文本、语音、图像)
- [x] 外部工具使用(工具体系)
- [ ] 长/短期记忆(话题、历史会话/文件) https://github.com/lobehub/lobe-chat/discussions/403
- [ ] 外挂知识库
- [ ] 主/被动交互

这些则是LobeChat 会去逐步实现的关键点。

----

https://zhuanlan.zhihu.com/p/656676717

大脑包括： 1）自然交互与意图理解 2）知识体系 3） 记忆 4） 推理

感知包括：1）文本识别 2） 视觉识别 3）音频识别

行动包括： 1）多模态输出 2）工具使用 3）具身行动

**Comment by MapleEve** [comment]
目前有一个已经落地的项目可参考
https://docs.deepwisdom.ai/main/zh/guide/tutorials/concepts.html

按照他的智能体概念是可以实现的，但是他的搭建还是需要python的能力，而我们目前的 Agent 其实做好重构之后就可以来进行类似的操作了
